src/Filter/components/FilterSectionHeader/FilterSectionHeader.tsx

```diff
@@ -1,7 +1,7 @@
 import { FC, MouseEventHandler } from 'react';
 import { FilterModule } from 'src/Filter/types';
-import { ReactComponent as Add } from 'src/Filter/assets/add.svg';
-import { ReactComponent as Remove } from 'src/Filter/assets/remove.svg';
+import { ReactComponent as Add } from '../../assets/add.svg';
+import { ReactComponent as Remove } from '../../assets/remove.svg';
```

Comment:
Can you add this to the contributing section of the README somewhere that makes sense? Would be good for future devs to know.

Reply:
Sure, I'll do that now. Good idea.